### DISCUSSION TITLE
Cubit Mesh

### DISCUSSION THREAD

**Discussion by Meghasemi-96** [idaholab/moose#25839] [Q&A Meshing]
Hello, 

I generated a simple mesh using Cubit containing a sphere at the center of a cubic box. Both of them have ID blocks (e.g., 0 and 1). When I solve my governing equations in the media, although I include both ID blocks in the MOOSE script, properties for one block are not considered for solvation.

Mesh generated by Cubit:
[1.zip](https://github.com/idaholab/moose/files/13125791/1.zip)

I was wondering how I can solve this issue.

Thanks, 
Mehdi

**Comment by GiudGiud** [comment]
Hello

What do you mean by "not considered for solvation" ?
do you get an error?

Guillaume

**Reply by Meghasemi-96** [reply]
Hello, 

Thanks for your prompt response. I have two different ID blocks and two different variables for solving fick's equation. I need to consider one particular diffusion for each variable in each domain, which means that I have to define 4 different diffusion coefficients. For example,

D_u in block 0 
D_u in block 1
D_v in block 0 
D_v in block 1

When I checked the output, I found that the inner block (i.g., 0) has no value for either of the variables. 
![output](https://github.com/idaholab/moose/assets/143123039/d89c5b20-aa99-4578-ac32-7681f604c852)

**Reply by GiudGiud** [reply]
u has no value, but D_u probably does.
you can use the material property output to check

If u has no value, it s probably because the mesh is disjoint between the two domains so information cannot go through. You ll need to merge nodes on the boundary in Cubit

**Reply by Meghasemi-96** [reply]
Thank you, I will check and come back to you.
